### PR TITLE
Update install.rst

### PR DESCRIPTION
**Pull request by cmcantalupo**
- Add instructions for all of the packages
- Clarify purpose of each package

### CODE REVIEW

File: service/docs/source/install.rst [hunk 1]
```diff
@@ -9,10 +9,14 @@ Build Service.
 
 The packages built from this service with each update are:
```
Comment (sidjana) [inline service/docs/source/install.rst]: To help avoid confusion w/ the overloaded term 'service', replace "built from this service with... " --> "built from OBS with..."
Also, add "(OBS)" at the end of the last sentence in the previous paragraph.

File: service/docs/source/install.rst [hunk 2]
```diff
@@ -5,14 +5,18 @@ Guide to Installation
 Continuous integration creates packages for a variety of Linux
```
Comment (dannosliwcd) [inline service/docs/source/install.rst:5]: Should this page be retitled to _Guide to Service Installation_? If not, should we add information about installing the runtime and geopmpy?

File: service/docs/source/install.rst [hunk 3]
```diff
@@ -56,3 +68,32 @@ least as recent as the release candidate branch.  For this reason,
 updates will always come from the development branch download
 repository if both the development repository and the release
 candidate repository are added to your package configuration,
+
+
+Un-packaged Features
+--------------------
+
+The installing the packages described above will not provide the
```
Comment (dannosliwcd) [inline service/docs/source/install.rst]:
```suggestion
The packages described above do not provide the
```

File: service/docs/source/install.rst [hunk 4]
```diff
@@ -56,3 +68,32 @@ least as recent as the release candidate branch.  For this reason,
...
+:doc:`GEOPM HPC Runtime <runtime>` features (e.g.
+:doc:`geopmlaunch(1) <geopmlaunch.1>`, :doc:`geopm_prof(3) <geopm_prof.3>`,
+:doc:`geopm_report(7) <geopm_report.7>`, ``libgeopm.so``, etc.).
+The GEOPM HPC Runtime must be built against a particular implementation of the
+Message Passing Interface (MPI).  Typically the implementation of MPI is
+specific to the system where the HPC application is being run.  An exception
+to this rule is the OpenHPC distribution.  GEOPM version 1 has been packaged
+with OpenHPC, and we hope to distribute version 2 with OpenHPC in the future.
```
Comment (dannosliwcd) [inline service/docs/source/install.rst]: Their download page has two release tracks. I think we're in both. If not, maybe specify which.

```suggestion
with `OpenHPC releases <http://openhpc.community/downloads/>`_, and we hope to distribute version 2 with OpenHPC in the future.
```

**Review by bgeltz — DISMISSED** [review]
With suggested changes

**Review by bgeltz — APPROVED** [review]

**Pull request merged**